doc/semieunit.xml

```diff
@@ -0,0 +1,292 @@
+
+      Finally, the user may use <C>McAlsiterTriple</C> with a single argument,
+      <A>S</A>, which is an E-unitary inverse semigroup and it will return a
+      McAlister triple semigroup that is isomorphic to <A>S</A>.
+  </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="IsMcAlisterTripleSemigroupElement">
+  <ManSection>
+    <Prop Name = "IsMcAlisterTripleSemigroupElement" Arg = "x"/>
+    <Prop Name = "IsMTE" Arg = "x"/>
+    <Returns><K>true</K> or <K>false</K>.</Returns>
+    <Description>
+      A <E>McAlister triple semigroup element</E> of the McAlister triple
+      semigroup <C>M:=McAlisterTripleSemigroup=(G,act,X,Y)</C> is a pair
+      consisting of a vertex <A>A</A> of the semilattice <C>Y</C> and an
+      element <A>g</A> of the group <C>G</C> such that 
+      <C>OnPoints(<A>A</A>,<A>g</A>^-1)</C> is in <C>Y</C>. The functions
+      <Ref Prop= "IsMTE"/> and <Ref Prop= "IsMcAlisterTripleSemigroupElement"/>
+      are identicial.
+
+<Example><![CDATA[
+gap> x := Digraph([[1], [1, 2], [1, 2, 3], [1, 4], [1, 4, 5]]);
+<digraph with 5 vertices, 11 edges>
+gap> y := InducedSubdigraph(x, [1, 2, 3]);
+<digraph with 3 vertices, 6 edges>
+gap> A := AutomorphismGroup(x);
+Group([ (2,4)(3,5) ])
+gap> S := McAlisterTripleSemigroup(A, OnPoints, x, y);
+<McAlister triple semigroup over Group([ (2,4)(3,5) ])>
+gap> T := McAlisterTripleSemigroup(A, x, y);
+<McAlister triple semigroup over Group([ (2,4)(3,5) ])>
+gap> T = S;
+true
+gap> a := MTE(S, 1, (2, 4)(3, 5));
+(1, (2,4)(3,5))
+gap> a * a;
+(1, ())
+gap> gens := GeneratorsOfSemigroup(S);
+[ (1, ()), (2, ()), (3, ()), (1, (2,4)(3,5)) ]
+gap> P := AsSemigroup(IsPartialPermSemigroup, S);
+<inverse partial perm monoid of size 4, rank 4 with 4 generators>
+gap> IsomorphismMcAlisterTripleSemigroup(P);
+MappingByFunction( <inverse partial perm monoid of size 4, rank 4 
+ with 4 generators>, <McAlister triple semigroup over Group([ (), (),
+ (1,2) ])>, function( s ) ... end )
+]]></Example>
+
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="McAlisterTripleSemigroupElement">
+  <ManSection>
+    <Oper Name = "McAlisterTripleSemigroupElement" Arg = "S, A, g"/>
+    <Oper Name = "MTE" Arg = "S, A, g"/>
+    <Returns>A McAlister triple semigroup element.</Returns>
+    <Description>
+      Returns the <E>McAlister triple semigroup</E> element of the McAlister
+      triple semigroup <A>S</A> which corresponds to the semilattice vertex
+      <A>A</A> and group element <A>g</A>, if this specifices an element of
+      <A>S</A>. The functions <Ref Prop= "MTE"/> and <Ref Prop= 
+        "McAlisterTripleSemigroupElement"/> are identicial.
+
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="AsMcAlisterTripleSemigroup">
+  <ManSection>
+    <Oper Name = "AsMcAlisterTripleSemigroup" Arg = "S"/>
+    <Returns>A McAlister triple semigroup.</Returns>
+    <Description>
+      This function can be used to find a McAlister triple semigroup which is
```

Comment:
I have just realised. `IsomorphismMcAlisterTripleSemigroup` should by implemented as `IsomorphismSemigroup` and I think `AsSemigroup` with filter `IsMcAlisterTripleSemigroup` will then work based on that. Does this sound right?